### PR TITLE
Disabled orators cannot be viewed or contacted 

### PR DESCRIPTION
**Pull request by khcr**
- The page of a disabled orator is not accessible anymore
- Cannot contact a disabled orator 
- The contact form has a working verification
- Add tests for the contact form.

### CODE REVIEW

File: features/orators/contact.feature [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    When I visit the orator's page
+    And I complete the contact form
+    Then I should see a flash with "Ton message a été envoyé. Tu as reçu une confirmation à l'email indiqué."
+    And the orator should receive an email
+
+  Scenario: I send a message with a wrong verification
+    Given there is a visible orator
+    When I visit the orator's page
+    And I complete the contact form with a wrong verification
+    Then I should see a flash with "Des champs sont invalides"
```
Comment (nkcr) [inline features/orators/contact.feature:18]:
```suggestion
    Then I should see a flash with "La vérification a échouée"
```

**Review by nkcr — APPROVED** [review]
Thanks! Minor comment about the error message. Not sure where it comes from.

File: features/orators/contact.feature (hunk 1 above)
Comment (khcr) [reply features/orators/contact.feature:18]: It is not sufficient to change the text in the test, you must also change the underlying code.

**Comment by khcr** [comment]
I display the errors in the contact form.
@nkcr can you design the errors ?

**Review by nkcr — APPROVED** [review]

**Pull request merged**